app/components/ChangeLog.tsx

```diff
@@ -88,15 +90,16 @@ export default function ChangeLog({ logs }: { logs: ChangeLogEntry[] }) {
 
   return (
     <Accordion
-      defaultActiveKey={expandedItem}
+      activeKey={expandedItem}
       // on accordion click save new expanded item state to local storage
-      onSelect={(e) => {
-        const localStorageValue = e !== null ? "0" : JSON.stringify(null); // local storage value must be a string type
+      onSelect={(eventKey) => {
+        const localStorageValue =
+          eventKey !== null ? recordHistoryItemName : String(null); // local storage value must be a string type
```

Comment:
nbd, but you can do away with the ternary and use  `String(eventKey)` as the `localStorageValue` since it will always be null or the `recordHistoryItemName`.